upgrading/topics/keycloak/changes-18_0_0.adoc

```diff
@@ -18,3 +18,41 @@ the migration. Consider these possible actions:
 insert into CLIENT_SCOPE_CLIENT (CLIENT_ID, SCOPE_ID, DEFAULT_SCOPE) select CLIENT.ID as CLIENT_ID, CLIENT_SCOPE.ID as SCOPE_ID, true as DEFAULT_SCOPE
 from CLIENT_SCOPE, CLIENT where CLIENT_SCOPE.REALM_ID='test' and CLIENT_SCOPE.NAME='acr' and CLIENT.REALM_ID='test' and CLIENT.PROTOCOL='openid-connect';
 ```
+
+= OpenID Connect Logout
+
+Previous versions of {project_name} had support for automatic logout of the user and redirecting to the application by opening logout endpoint URL like
+`https://auth-server/auth/realms/{realm-name}/protocol/openid-connect/logout?redirect_uri=encodedRedirectUri`. This was easy to use, but had potentially negative impact
+on performance and security. The new version has proper support for logout based on the OpenID Connect RP-Initiated Logout specification. The parameter `redirect_uri` is not
+supported anymore and user needs to confirm the logout. It is possible to omit the confirmation and do automatic redirect to the application when you include parameter `post_logout_redirect_uri`
+together with the parameter `id_token_hint` with the ID Token used for login.
+
+The existing deployments are affected as described below:
+
+- If your application directly uses links to logout endpoint with the `redirect_uri` parameter, you may be required to change this as described above.
```

Comment:
use "log out" when it is a verb -  "log out an endpoint"

Reply:
This is actually not the verb. The "logout endpoint" is the name of the endpoint. It is the REST endpoint, which is used for logout user and we call it "Logout endpoint". So for now, I keep this part.

For more illustration, see for example this part of the existing documentation https://www.keycloak.org/docs/latest/securing_apps/index.html#logout and the last sentence: This is useful when the logout endpoint is invoked as part of single logout initiated by the external identity provider.

Reply:
Okay, thanks for the clarification, @mposolda.